lib/views/item/item-detail.js

```diff
@@ -180,12 +184,60 @@ class ItemDetail extends Component {
       <View style={styles.distance}>
         <Image
           style={styles.distanceImage}
-          source={require('../../images/item-scene-room.png')}/>
+          source={{
+            uri: 'ic_place',
```

Comment:
Is there a reason why this image is not loaded using the regular `require()` mechanism?